public/locales/en/common.json

```diff
@@ -185,7 +185,7 @@
   "optional": "Optional",
   "Page next": "Next page",
   "Page not found": "Could not find the page",
-  "Page numbers": "Page numbers",
+  "Page numbers": "Pagination",
```

Comment:
?

Reply:
This text is the screen-reader label for the pagination widget. "Pagination" sounds good though.